### PR TITLE
upgrade to ddf-2.24.0

### PR DESCRIPTION
**Pull request by glenhein**
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)
#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)

@andrewkfiedler
@bdeining
@beyelerb
@brendan-hofmann
@brjeter
@clockard
@coyotesqrl
@figliold
@jaymcnallie
@jlcsmith
@kcwire
@lessarderic
@michaelmenousek
@millerw8
@mrmateo
@pklinef
@ricklarsen - Documentation
@rzwiefel
@shaundmorris
@stustison
@tbatie
@vinamartin
#### What are the relevant tickets?

[CAL-xxx](https://codice.atlassian.net/browse/CAL-xxx)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by shaundmorris** [comment]
Thanks for putting up this PR @glenhein I wonder if we should reconsider keeping Alliance master up to date with DDF master.

**Comment by jlcsmith** [comment]
build now

**Comment by glenhein** [comment]
build now

**Comment by glenhein** [comment]
@shaundmorris we don't intend on moving alliance master to ddf snapshot. This PR is just to make sure there are no major issues with moving alliance master to the next ddf release.

**Comment by shaundmorris** [comment]
Thanks @glenhein got it. We used to keep snapshot on there so we knew the latest would work. But going the release model definitely has the pro of stability 👍

**Comment by glenhein** [comment]
build now

File: pom.xml [hunk 1]
```diff
@@ -120,7 +120,7 @@
         <yarn.version>v1.5.1</yarn.version>
 
         <!--Project properties-->
-        <ddf.version>2.23.1</ddf.version>
+        <ddf.version>2.24.0-SNAPSHOT</ddf.version>
```
Comment (glenhein) [inline pom.xml:123]: This will be changed to 2.24.0 once the ddf release is complete.

**Comment by jlcsmith** [comment]
build now

**Pull request merged**